Packs/AWS/Integrations/AWS/AWS.yml

```
@@ -1397,6 +1608,230 @@ script:
+        defaultValue: 'true'
+        hidden: true
+
+  - name: aws-rds-db-instance-enable-iam-auth-quick-action
+    description: Enables 'iam database authentication' for the RDS dB instance by running the aws rds modify-db-instance command in your environment. Successful execution requires 'rds:ModifyDBInstance' permission.
+    prettyname: Enable IAM Authentication on RDS
+    hidden: true
+    quickaction: true
+    arguments:
+      - name: account_id
+        description: The AWS account ID.
+        prettyname: Account ID
+        required: true
+        prettypredefined:
+          Account ID: ${asset.xdm.asset.realm}
+      - name: region
+        description: The AWS region.
+        prettyname: Region
+        required: true
+        prettypredefined:
+          Region: ${asset.xdm.asset.cloud.region}
+      - name: db_instance_identifier
+        prettyname: RDS Instance ID
+        description: 'The identifier of DB instance to modify.'
+        required: true
+        prettypredefined:
+          RDS Instance ID: ${asset.xdm.asset.name}
+      - name: enable_iam_database_authentication
+        description: 'Weather to enable mapping of Amazon Web Services Identity and Access Management (IAM) accounts to database accounts. By default, mapping isn’t enabled.'
+        defaultValue: 'true'
+        hidden: true
+
+  - name: aws-rds-db-instance-enable-deletion-protection-quick-action
+    description: Enables 'deletion protection' for the RDS dB instance by running the aws rds modify-db-instance command in your environment. Successful execution requires 'rds:ModifyDBInstance' permission.
+    prettyname: Enable Deletion Protection on RDS
+    hidden: true
+    quickaction: true
+    arguments:
+      - name: account_id
+        description: The AWS account ID.
+        prettyname: Account ID
+        required: true
+        prettypredefined:
+          Account ID: ${asset.xdm.asset.realm}
+      - name: region
+        description: The AWS region.
+        prettyname: Region
+        required: true
+        prettypredefined:
+          Region: ${asset.xdm.asset.cloud.region}
+      - name: db_instance_identifier
+        prettyname: RDS Instance ID
+        description: 'The identifier of DB instance to modify.'
+        required: true
+        prettypredefined:
+          RDS Instance ID: ${asset.xdm.asset.name}
+      - name: deletion_protection
+        description: 'Weather the DB instance has deletion protection enabled. The database can’t be deleted when deletion protection is enabled. By default, deletion protection isn’t enabled. For more information, see Deleting a DB Instance.'
+        defaultValue: 'true'
+        hidden: true
+
+  - name: aws-rds-db-instance-enable-auto-upgrade-quick-action
+    description: Enables automatic minor version upgrades for the RDS database instance by running the aws rds modify-db-instance command. Successful execution requires the rds:ModifyDBInstance permission.
```

```suggestion
    description: Enables automatic minor version upgrades for the RDS database instance by running the aws rds modify-db-instance command.
```
Replace aws rds modify-db-instance.